taxi/views.py

```diff
@@ -52,3 +51,37 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
```

Comment:
`__all__` is a default option, there is no need to add it in the future without necessity

Reply:
I have tryed to delete this field, but I get next error:  Using ModelFormMixin (base class of CarCreateView) without the 'fields' attribute is prohibited.